DTS-STN/passport-status
# Added tooltips to the email and status forms

## [ADO-1223](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1223) & [ADO-1224](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1224)

### Description
This PR adds tooltips based on the form text pattern seen [here](https://getbootstrap.com/docs/5.2/forms/overview/#form-text). As per our discussions during standup this provides the most accessible experience for the user vs a pop up tooltip.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Fire up your favorite screen reader (NVDA, JAWS, etc.)
4. Navigate to both `/status` and `/email` and ensure the tooltips are both visible and announced by the screen reader
